src/pages/common/AuthFormComponent/RegistrationFormComponent.tsx

```diff
@@ -0,0 +1,99 @@
+import React from "react";
+import { useAppDispatch } from "../../../hooks/useAppDispatch";
+import { Formik, Form, Field } from "formik";
+import "./style.css";
+import * as yup from "yup";
+import { registration } from "../../../store/reducers/actionCreator";
+import { useAppSelector } from "../../../hooks/useAppSelector";
+import Loader from "../../../components/Loader/Loader";
+
+const RegistrationFormComponent = () => {
+  const dispatch = useAppDispatch();
+  const { loading, errorRegistration } = useAppSelector(
+    (state) => state.authReducer
+  );
+
+  const Registration = (
+    name: string,
+    surname: string,
+    email: string,
+    password: string
+  ) => {
+    dispatch(registration({ name, surname, email, password }));
+  };
+
+  const initialValues = {
```

Comment:
Const values and types better move outside of the component to prevent rerendering and reinitialization